docs/en/concepts-and-designs/event.md

```diff
@@ -13,6 +13,7 @@ Currently, the officially supported clients to report events are:
 - [ ] Java Agent Toolkit: Using the Java agent toolkit to report events within the applications.
 - [x] SkyWalking CLI: Using the CLI to report events from the command line interface.
 - [x] [Kubernetes Event Exporter](http://github.com/apache/skywalking-kubernetes-event-exporter): Deploying an event exporter to refine and report Kubernetes events.
+- [x] HTTP Client: Using the HTTP Client to report events within the applications. 
```

Comment:
This is kind of misleading, this list is "clients" but we don't provide "HTTP client" for event reporter. This PR only accepts event data from HTTP protocol, itself doesn't provide an HTTP client to report events.

I think line 10 has already covered the http protocol.

> The SkyWalking backend supports three protocols to collect events: gRPC, HTTP, and Kafka. Any agent or CLI that implements one of these protocols can report events to SkyWalking.